api-functions/CMS/Contentful.ts

```diff
@@ -0,0 +1,271 @@
+export const CMS_ERRORS = {
+  unableToFetch: new Error("unable to fetch content from CMS"),
+  itemsUndefined: new Error("cms failed to return expected items"),
+  metadataNotUnique: new Error(
+    "Metadata already defined. Only one metadata field can be present"
+  ),
+  missingData: (field: string) =>
+    new Error(`cms failed to return value for field: ${field}`),
+};
+export type LandingMetadata = {
+  title: string;
+  description: string[];
+  buttonStartNow: string;
+  whyThisToolTitle: string;
+  howWeHelpTitle: string;
+  kitchenIntro: string;
+  didYouKnowTitle: string;
+  articleTitle: string;
+};
+
+export type LandingExampleViolation = {
+  title: string;
+  description: string;
+  violationReason: string;
+  order: number;
+  markerLeftPosition: number;
+  markerTopPosition: number;
+};
+
+export type LandingFact = {
+  title: string;
+  description: string;
+  backgroundImage: string;
+  order: number;
+  lightTextColor: boolean;
+};
+
+export type LandingExplanation = {
+  title: string;
+  description: string;
+  hyperlink?: { uri: string; text: string };
+  order: number;
+};
+
+export type RichElement = {
+  nodeType?: string;
+  value?: string;
+  subRichElements?: SubRichElement[];
+  order: number;
+  icon: string;
+};
+
+export type SubRichElement = {
+  nodeType: string;
+  value: string;
+  uri?: string;
+};
+
+export type DidYouKnow = {
+  title: string;
+  description: string;
+  sourceName: string;
+  sourceUrl: string;
+  order: number;
+};
+export type RelevantArticle = {
+  title: string;
+  sourceName: string;
+  sourceUrl: string;
+  order: number;
+  previewImage: string | null;
+};
+
+export type LandingContent = {
+  metadata: LandingMetadata;
+  violations: LandingExampleViolation[];
+  facts: LandingFact[];
+  explanations?: RichElement[];
+  didYouKnows: DidYouKnow[];
+  relevantArticles: RelevantArticle[];
+};
+
+const url =
+  "https://cdn.contentful.com/spaces/56c95v53ajrr/environments/master/entries?access_token=WCe8q5WXL2WA0btELHrTuVAOtSgKXvK7lIFUlYg1Xqo&metadata.tags.sys.id[in]=landing";
```

Comment:
Please take greater care with secret keys in the future. By committing the secret key and pushing it to Github you have exposed the key to the entire internet which defeats the purpose of secret keys. In this case it isn't too detrimental since the key only gives access to the delivery api. However, if this was a Database salt or something this would be a lot more troubling.

I have already gone ahead and rolled the keys on Vercel. Please remove this key on your end to deactivate it.